hexapod_controller:
  ros__parameters:
    dt: 0.01
    pid:
      bias:
        Ax: 0.1623966943359376
        Ay: 0.21198796875000012
        Az: 0.3021586914062535
        Gx: 0.24914503816793732
        Gy: 0.5329618320610683
        Gz: 0.0
        calibrated: 0
        count: 500
        pitch: 0.978634914347575
        roll: 1.2775255765111633
      pitch:
        filter_coe: 0.95
        kd: 0.03
        ki: 0.1
        kp: 1.5
        max_I: 40
        max_angle: 60
      roll:
        filter_coe: 0.95
        kd: 0.03
        ki: 0.1
        kp: 1.5
        max_I: 40
        max_angle: 60
      yaw:
        kd: 0
        ki: 0
        kp: 1
        max_I: 10
        max_angle: 20
      
